presto-phoenix/pom.xml

```diff
@@ -311,6 +311,12 @@
                         <!-- io.airlift:joni and phoenix-client's org.jruby.joni:joni resource duplicates-->
                         <ignoredResourcePattern>tables/.*\.bin</ignoredResourcePattern>
                     </ignoredResourcePatterns>
+                    <ignoredDependencies>
+                        <dependency>
+                            <groupId>com.clearspring.analytics</groupId>
+                            <artifactId>stream</artifactId>
+                        </dependency>
+                    </ignoredDependencies>
```

Comment:
Why do we need this?

Reply:
Without this exclusion, the compilation fails. Apache Phenix seems to depend on [the stream library](https://github.com/apache/phoenix/blob/7e955ecbcf967e74fbb37e5d0c9c5556ebac5898/pom.xml#L1317-L1321). 

```
[ERROR] Failed to execute goal org.basepom.maven:duplicate-finder-maven-plugin:1.4.0:check (default) on project presto-phoenix: Found duplicate classes/resources! -> [Help 1]
```

Reply:
But why does this connector depend on presto-main? Or is it just for tests?

Reply:
Yes. Phenix connector seems to depend on presto-main with test scope.

Reply:
@electrum, any thoughts on how to handle this? Adding an exclusion to the duplicate finder doesn't seem ideal. I guess the underlying problem is the fact that the connector depends on presto-main, even if it's for tests. I know we do it everywhere, but this highlights the problem with that pattern.

Reply:
This looks like a problem with the Phoenix shading. They are including the classes but not repackaging them. (just a guess, I haven’t looked at the failure in detail)